changelog/allow_casting_from_typetuple_to_typetuple.dd

```diff
@@ -0,0 +1,15 @@
+Allow casting from TypeTuple to TypeTuple
+
+It wan not allowed to cast from TypeTuple to TypeTuple even if they are the same type is.
```

Comment:
```suggestion
It was not allowed to cast from `TypeTuple` to `TypeTuple` even if they are the same type is.
```

Reply:
Again, TypeTuple should not be present in the changelog. Simply refer to them as builtin tuples.